### PR TITLE
[zh-CN]: sync translation for Notifications API & Battery Status API

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

~depends on https://github.com/mdn/content/pull/35901 and https://github.com/mdn/content/pull/35904~

### CODE REVIEW

File: files/zh-cn/web/api/notifications_api/index.md [hunk 1]
```diff
@@ -20,17 +20,12 @@ btn.addEventListener("click", () => {
...
 ![一个对话框，要求用户允许来自该源的通知。有一些选项可以禁止或允许通知](screen_shot_2019-12-11_at_9.59.14_am.png)
 
 从这里，用户可以选择允许来自此来源的通知或阻止来自此来源的通知。一旦做出选择，该设置通常将持续用于当前会话。
 
-> [!NOTE]
-> 从 Firefox 44 开始，通知（Notification）和推送（[Push](/zh-CN/docs/Web/API/Push_API)）的权限已合并。如果为通知授予权限，推送也将启用。
-
 接下来，使用 {{domxref("Notification.Notification","Notification()")}} 构造方法创建一个新通知。这个函数必须传递一个标题参数，并且可以选择传递一个选项对象来指定选项，例如文本方向、正文、要显示的图标、要播放的通知的声音等等。
 
 此外，Notifications API 规范对 [ServiceWorker API](/zh-CN/docs/Web/API/ServiceWorker_API) 指定了多个扩展，以允许 Service Worker 发送通知。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notifications_api/index.md:31]:
```suggestion
此外，Notifications API 规范对 [ServiceWorker API](/zh-CN/docs/Web/API/Service_Worker_API) 指定了多个扩展，以允许 Service Worker 发送通知。
```

File: files/zh-cn/web/api/notification/requestpermission_static/index.md [hunk 2]
```diff
@@ -2,20 +2,21 @@
...
 l10n:
-  sourceCommit: eb61adfd5ca85cd5cb2143353f82fe0520a9c66e
+  sourceCommit: 09ad551d5fecae5872328ece2871fdf02b115b6e
 ---
 
 {{APIRef("Web Notifications")}}{{securecontext_header}}
 
 {{domxref("Notification")}} 接口的 **`requestPermission()`** 静态方法向用户为当前来源请求显示通知的权限。
 
+该方法返回一个 {{jsxref("Promise")}}，该字符串表示是否授予或拒绝权限。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/requestpermission_static/index.md:12]:
```suggestion
该方法返回一个 {{jsxref("Promise")}}，会兑现一个表示是否授予或拒绝权限的字符串。
```

File: files/zh-cn/web/api/notification/requestpermission_static/index.md [hunk 3]
```diff
@@ -2,20 +2,21 @@
...
 ## 语法
 
 ```js-nolint
-// 最新规范已将此方法更新为基于 promise 的语法，其工作方式如下：
 Notification.requestPermission()
 
-// 以前，语法是基于一个简单的回调函数；此版本现已弃用：
+// 弃用的使用回调的语法
 Notification.requestPermission(callback)
 ```
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/requestpermission_static/index.md:22]: 26 行的 promise 不是链接

File: files/zh-cn/web/api/notifications_api/using_the_notifications_api/index.md [hunk 4]
```diff
@@ -201,20 +213,22 @@ window.addEventListener("load", () => {
...
   });
 });
 ```
 
 ### 结果
 
-{{ EmbedLiveSample('标签示例', '100%', 30) }}
+{{ EmbedLiveSample('标签示例', '100%', 200) }}
+
+为了测试上述示例，请更改 `https://live.mdnplay.dev` 网站的[发送通知设置](https://support.mozilla.org/zh-CN/kb/firefox-page-info-window#w_permissions)。
```
Comment (yin1999) [inline files/zh-cn/web/api/notifications_api/using_the_notifications_api/index.md:231]:
```suggestion
为了测试上述示例，请更改 `https://live.mdnplay.dev` 网站的[发送通知设置](https://support.mozilla.org/zh-CN/kb/页面信息窗口#w_quan-xian)。
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**